ThirdPartyNotices.md

```diff
@@ -9,7 +9,7 @@ may be either trademarks or registered trademarks of Microsoft in the United Sta
 The licenses for this project do not grant you rights to use any Microsoft names, logos, or trademarks.
 Microsoft's general trademark guidelines can be found at http://go.microsoft.com/fwlink/?LinkID=254653.
 
-Privacy information can be found at https://privacy.microsoft.com/en-us/
+Privacy information can be found at https://privacy.microsoft.com/en-us
```

Comment:
This should actually be a link to https://privacy.microsoft.com That way, the link redirects to the correct culture based on the browser's culture.